test/e2e/pages/editor.page.ts

```diff
@@ -50,7 +50,9 @@ export class EditorPage extends BasePage {
     playIconSelector: 'i.fa-play',
     dropdownToggleSelector: 'a.dropdown-toggle',
     uploadButtonSelector: 'button.upload-audio',
-    downloadButtonSelector: 'a.buttonAppend'
+    downloadButtonSelector: 'a.buttonAppend',
+    slider: '#slider',
```

Comment:
I'm pretty sure the test is failing since the id was removed
```suggestion
    slider: 'input.seek-slider',
```